objects/src/accounts/package/storage_entry/template.rs

```diff
@@ -0,0 +1,115 @@
+use alloc::{
+    string::{String, ToString},
+    vec::Vec,
+};
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use vm_core::{Felt, Word};
+use vm_processor::Digest;
+
+use crate::accounts::package::AccountComponentTemplateError;
+
+// TEMPLATE KEY
+// ================================================================================================
+
+// A simple wrapper type around a string key, used to enable templating.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct TemplateKey {
+    key: String,
+}
+
+impl TemplateKey {
+    pub fn new(key: impl Into<String>) -> Self {
+        Self { key: key.into() }
+    }
+
+    pub fn inner(&self) -> &str {
```

Comment:
```suggestion
    pub fn as_str(&self) -> &str {
```

Nit: This might be more idiomatic.

Reply:
Named it `inner` after Bobbin's suggestion, but I agree.